docs/src/main/asciidoc/features.adoc

```diff
@@ -30,6 +30,9 @@ latency in your applications. Sleuth is written to not log too much, and to not
 * Instruments common ingress and egress points from Spring applications (servlet filter, async endpoints,
 rest template, scheduled actions, message channels, zuul filters, feign client).
 
+* Via `SpanInjector` and `SpanExtractor` interface allows to customize the way spans are
```

Comment:
not sure if this is too long winded..

Sleuth includes default logic to join a trace across http or messaging boundaries. For example, http propagation works via Zipkin-compatible request headers. This propagation logic is defined and customized via `SpanInjector` and `SpanExtractor` implementations.